tests/wpt/meta/content-security-policy/embedded-enforcement/subsumption_algorithm-none.html.ini

```diff
@@ -1,21 +1,9 @@
 [subsumption_algorithm-none.html]
-  [Required policy that allows `none` does not subsume empty list of policies.]
-    expected: FAIL
-
-  [Required csp with effective `none` does not subsume a host source expression.]
-    expected: FAIL
-
   [Required csp with `none` does not subsume a host source expression.]
     expected: FAIL
 
-  [Required csp with effective `none` does not subsume `none` of another directive.]
+  [Both required and returned csp are `none` for only one directive.]
```

Comment:
Not sure with this one, but I think it's a similar test that was failing (e.g. a prerequisite)